mdn/developer-portal
# 1602+1603: Update Events page to have content options the top and and bottom

Thi changeset adds two custom streamfields, on at the top and one at the bottom of the main content area on the Events page.

This provides greater flexibility in adding additional signposting, etc to the page

Resolves #1602
Resolves #1603

## Key changes:

- Update `Events` page config, including database migration
- Update templates to use new fields

## How to test

- Code is deployed to staging
- Edit the Events page to add things, including buttons
  - Note that buttons-as-links have a large amount of bottom margin (100px) so using regular button elements in the streamfield is preferable 


## Example screenshots

![Screenshot_2020-06-29 Events](https://user-images.githubusercontent.com/101457/86002904-f029d700-ba08-11ea-80ad-abfa45f4406e.png)
![Screenshot_2020-06-29 Events(1)](https://user-images.githubusercontent.com/101457/86002919-f5872180-ba08-11ea-8dfc-5b4f7f982a9f.png)
